core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -140,6 +140,12 @@
      */
     private transient int numRetryAttempt;
 
+    /**
+     * Escape hatch for rejecting connections from agents with unsupported Remoting versions
+     */
+    @Restricted(NoExternalUse.class)
+    public static boolean REJECT_CONNECTION = SystemProperties.getBoolean(SlaveComputer.class.getName() + ".allowUnsupportedAgentConnection");
```

Comment:
```suggestion
    private static boolean REJECT_CONNECTION = SystemProperties.getBoolean(SlaveComputer.class.getName() + ".allowUnsupportedAgentConnection");
```

Otherwise you will be hit by @jtnord ;)